velox/expression/CastExpr-inl.h

```diff
@@ -285,6 +285,27 @@ void CastExpr::castTimestampToDate(
   });
 }
 
+template <typename IntType>
+void CastExpr::castIntegerToTimestamp(
+
+    const SelectivityVector& rows,
+    const BaseVector& input,
+    exec::EvalCtx& context,
+    VectorPtr castResult,
+    const facebook::velox::date::time_zone* timeZone,
+    FlatVector<Timestamp>* resultFlatVector) {
+  applyToSelectedNoThrowLocal(context, rows, castResult, [&](int row) {
+    auto timestamp = Timestamp::fromMillis(
+        static_cast<int64_t>(input.as<SimpleVector<IntType>>()->valueAt(row)) *
+
+        kMillisInSecond);
```

Comment:
In find in Spark, `SECONDS.toMicros(t)` is implemented as below where `x` will check the long value against a maximum value. Do we need similar implementation in Velox?
```
   public long toMicros(long var1) {
            return x(var1, 1000000L, 9223372036854L);
        }

    static long x(long var0, long var2, long var4) {
        if (var0 > var4) {
            return Long.MAX_VALUE;
        } else {
            return var0 < -var4 ? Long.MIN_VALUE : var0 * var2;
        }
    }
```